nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -469,6 +469,211 @@ def _impl(inputs, attr, params):
         return inputs[0]
     return _impl
 
+def _fill():
+    def _impl(inputs, attr, params):
+        fill_arg = params.pop(inputs.pop(1).list_output_names()[0])
+        new_inputs = []
+        return AttrCvt(
+            op_name='full',
+            extras={'shape':inputs[0],
+                    'fill_value':fill_arg.asnumpy()[0], 'dtype':attr['T'].name},
+            ignores=['index_type', 'T'])(new_inputs, attr)
+    return _impl
+
+def _gather_v2():
+    "Tensorflow now support only gatherv2"
+    def _impl(inputs, attr, params):
+        axis = params[inputs.pop(2).list_output_names()[0]].asnumpy()[0]
+        new_input = []
+        new_input.append(inputs.pop(0))
+        new_input.append(inputs.pop(0))
+        return AttrCvt(
+            op_name="take",
+            extras={'axis':axis},
+            ignores=['Tindices', 'Tparams', 'validate_indices', \
+                     'Taxis', '_class'])(new_input, attr)
+    return _impl
+
+def _infer_out_shapes(inputs, params):
+    """A method to get the output shape of an intermediate node in the NNVM graph."""
+    g = _graph.create(inputs)
+    shape_dict = {k: v.shape for k, v in params.items()}
+    _, out_shapes = graph_util.infer_shape(g, **shape_dict)
+    return out_shapes
+
+def _stridedSlice():
+    def _impl(inputs, attr, params):
+        """Strided Slice.
+        Description:
+        https://www.tensorflow.org/api_docs/python/tf/strided_slice
+
+        Parameters
+        ----------
+        inputs : nnvm.Symbol
+            Input data
+        attrs : dict
+            Dict of operator attributes
+        params : dict
+            List of pretrained weights and bias
+
+        Returns
+        -------
+        sym : nnvm.Symbol
+            Converted nnvm Symbol
+        """
+        begin_input = params.pop(inputs[1].list_output_names()[0]).asnumpy().tolist()
+        end_input = params.pop(inputs[2].list_output_names()[0]).asnumpy().tolist()
+        stride_input = params.pop(inputs[3].list_output_names()[0]).asnumpy().tolist()
+        begin_mask = int(attr.get('begin_mask', 0))
+        end_mask = int(attr.get('end_mask', 0))
+        ellipsis_mask = int(attr.get('ellipsis_mask', 0))
+        new_axis_mask = int(attr.get('new_axis_mask', 0))
+        shrink_axis_mask = int(attr.get('shrink_axis_mask', 0))
+
+        #Constant values used forming output shape.
```

Comment:
Thanks for cleaning the above logic.
Would suggest to clean the below part as well a bit.

My understanding of all these masks as below. Please correct me if I am wrong.

begin, end: Basically to ignore a value from the inputs and use 0 for begin, max for end.
elipsismask: Only one bit will be set which is used to expand begin, end and strides to size of input dimensions by filling max range in between.
newaxis mask: add new axis in the result based on the mask bits.
shringaxis_mask: shrink the axis from result based on the mask bits.

I see this logic can be split into below logically separable blocks.
1: Handle begin and end masks on begin, end
2: Expand begin and end based on elipsis_mask
3: Apply strided_slice operation
4: Apply reshape operation with newaxis and shrink axis 

Also suggest to use python API for converting bitmask into lists of integers.